components/invoice.js

```diff
@@ -48,43 +60,31 @@ export default function Invoice ({ invoice, modal, onPayment, info, successVerb,
     variant = 'failed'
     status = 'expired'
     webLn = false
+  } else if (invoice.expiresAt) {
+    variant = 'pending'
+    status = (
+      <CompactLongCountdown
+        date={invoice.expiresAt} onComplete={() => {
+          setExpired(true)
+        }}
+      />
+    )
```

Comment:
Replace loading spinner + expiration countdown to be just the expiration countdown.